src/components/BlockingViews/FullPageNotFoundView.js

```diff
@@ -64,6 +64,7 @@ function FullPageNotFoundView({children, shouldShow, titleKey, subtitleKey, link
                 <View style={[styles.flex1, styles.blockingViewContainer]}>
                     <BlockingView
                         icon={Illustrations.ToddBehindCloud}
+                        iconColor={undefined}
```

Comment:
Instead of passing `undefined` here, I think we should set the default prop value for the `BlockingView` `iconColor` to `undefined`.

Reply:
It's connected with the issue mentioned above. Value undefined indicates that we want to display svg icon in its default color. I've attached the comparison between two blocking views displayed with and without passed undefined value. If we don't pass iconColor={undefined} icon is displayed in `themeColors.offline` color. Solution which can make code more readable is setting default value of iconColor to undefined, and pass `themeColors.offline` to components when it's necessary
![Simulator Screenshot - iPhone 15 - 2023-11-14 at 11 18 27](https://github.com/Expensify/App/assets/47774969/099e1fd5-5b66-4bd5-9059-7d49c50d9637)
![Simulator Screenshot - iPhone 15 - 2023-11-14 at 11 18 54](https://github.com/Expensify/App/assets/47774969/50c88786-409e-4fe1-acbf-9d1fd7e79923)

Reply:
> Solution which can make code more readable is to set the default value of iconColor to undefined and pass themeColors.offline icon color to components when it's necessary. Will it solve the problem?

Yes, I think we should do this. This would also apply to the other instances I raised in the review. Thanks.